ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -5,12 +5,15 @@ metadata:
   name: operator-hosted-pipeline
 spec:
   params:
+    # When adding a new param, make sure to also add it to the webhook in
```

Comment:
There wasn't really a good place in the README, so I decided to add a comment in the pipeline file instead. It's probably more visible this way.